### ISSUE TITLE
How can I get the unit test passed with the 3.1.1 (a94bb128faf1a0fd54303f2223415ee65268e8bd)?

### ISSUE THREAD

**Issue by SHI-ZP** [aws-solutions/distributed-load-testing-on-aws#114]
Hi,

I am doing some customization work for the Taurus images, based on the commit a94bb128faf1a0fd54303f2223415ee65268e8bd.

But I can not get the unit tests passed on my mac:
![image](https://user-images.githubusercontent.com/16377386/205653573-ac922808-bdb5-4185-9a07-d24ab36be93a.png)

It looks like the major blocker is the module "nanoid" can not be found by node:
![image](https://user-images.githubusercontent.com/16377386/205654394-2ea11c80-a540-4e2c-93e9-b10028b1276b.png)

I just followed the steps in the readme. And my nodejs version is v19.2.0.

Many thanks for the help.

**Comment by G-Lenz** [comment]
Hi @SHI-ZP 
It seems nanoid was not included in the solution-utils, most likely because solution-utils dependencies were not yet installed. You can fix this by going into the source/solution-utils folder then running `npm install`. Once the packages are installed you can go to the source/api-services folder and run `npm run clean` followed by `npm install` to re-install the dependencies for it.

**Comment by NEC-Maeta** [comment]
Is it happening when running run-unit-tests.sh?
It happened in my environment, but the AWS official documentation does not include instructions for running this script.
For this reason I skipped.

Also during subsequent build-s3-dist.sh execution (npm install on line 62)
"npm ERR! ERESOLVE unable to resolve dependency tree" occurred.
It's probably a dependency, but I added "--legacy-peer-dep" to deal with it.
